src/UI/examples/Popover/show_popover_with_async_loaded_content.php

```diff
@@ -0,0 +1,22 @@
+<?php
+function show_popover_with_async_loaded_content()
+{
+	global $DIC;
+	$factory = $DIC->ui()->factory();
+	$renderer = $DIC->ui()->renderer();
+
+	if (isset($_GET['renderPopoverAsync']) && $_GET['renderPopoverAsync']) {
```

Comment:
Add a short description of what is done here. This is part of the documentation and must be very clear.